app/src/pages/admin/serverSettingsPage/serverSettingsPage.jsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
-import { FormattedMessage } from 'react-intl';
-import { PageLayout } from 'layouts/pageLayout';
+import React, { Component } from 'react';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { PageLayout, PageHeader, PageSection } from 'layouts/pageLayout';
 import { ServerSettingsTabs } from './serverSettingsTabs';
 
-export const ServerSettingsPage = () => (
-  <PageLayout
-    title={<FormattedMessage id={'ServerSettingsPage.title'} defaultMessage={'Server settings'} />}
-  >
-    <ServerSettingsTabs />
-  </PageLayout>
-);
+const messages = defineMessages({
+  pageTitle: {
+    id: 'ServerSettingsPage.title',
+    defaultMessage: 'Server settings',
+  },
+});
+
+@injectIntl
+export class ServerSettingsPage extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  render() {
+    return (
+      <PageLayout>
+        <PageHeader
+          breadcrumbs={[
```

Comment:
Just a little suggestion: make method `getBreadcrumbs` which returns this array. Just to make JSX look less bloated.

Reply:
Done